packages/flutter_tools/lib/src/windows/build_windows.dart

```diff
@@ -53,6 +54,7 @@ Future<void> buildWindows(WindowsProject windowsProject, BuildInfo buildInfo, {
 
   final List<ProjectMigrator> migrators = <ProjectMigrator>[
     CmakeCustomCommandMigration(windowsProject, globals.logger),
+    CmakePolicyMigration(windowsProject, globals.logger),
```

Comment:
Linux needs this migration too, right? If so, please update: https://github.com/flutter/flutter/blob/ef82f97fc0417e5bc7786b5308279a597d15ccdc/packages/flutter_tools/lib/src/linux/build_linux.dart#L46